OpenRA.Mods.Common/Activities/DonateCash.cs

```diff
@@ -34,7 +34,7 @@ protected override void OnInside(Actor self)
 			if (target.IsDead)
 				return;
 
-			target.Owner.PlayerActor.Trait<PlayerResources>().GiveCash(payload);
+			target.Owner.PlayerActor.Trait<PlayerResources>().ChangeCash(payload);
```

Comment:
This will need to be `payload = target.Owner.PlayerActor.Trait<PlayerResources>().ChangeCash(payload);` so that the CashTick can show the correct value.

Reply:
Payload is readonly. Are you sure you want me to change it?

Reply:
Ah ok, better to define a new `donated` variable then.

Reply:
Ok. I'm on it.